templates/email/quote-expired.hypr

```diff
@@ -0,0 +1,25 @@
+﻿{% extends "email/email" %}
+{% block body-content %}
+         <!--- Header --->
+            <div class="mz-curbside-email-header">  
+            <img src="https://kibocommerce.com/wp-content/themes/kibo/images/logo.png" alt="logo">
+            <div class="mz-curbside-header-body">
+                <p><strong>{{ siteContext.generalSettings.websiteName }}</strong></p>
+                <h3>{{ labels.curbsideOrderConfirmationtxt }}</h3>
+            </div>
+            </div>
+           
+<div style="margin: 3% 0;">
```

Comment:
Remove inline styling and create classes and use it here. Do this for all templates.

Reply:
Remove inline css and added <br/> tab for manage text content space